### PR TITLE
Assessor views submitted application questions/answers

### PR DESCRIPTION
**Pull request by Harriethw**
# Context

https://trello.com/c/OMQN3QPp/588-nacro-submitted-application-answers
We would like to display the questions and answers of a submitted application to assessors.
The design will match the content rendered in the Check Your Answers page, but will need to be generated from the Application's `document` field (which is the canonical source of truth for the content of a submitted application)

# Changes in this PR

This introduces a new macro/partial to generate the Summary List from the `document` sections and adds it the `show` template for applications for assessors. 
I will shortly be introducing the same macro the [submitted application view for referrers](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/pull/280) as the code is so similar.

I'm a bit concerned about the integration test that I've written as it's quite slow (similar to the CYA test), but not sure how else we'd go about it.
The best way of manually testing this is grabbing the ID of a submitted application, then to login as CAS2_ASSESSOR and go to `http://localhost:3000/assess/applications/{id}`

## Screenshots of UI changes
Screenshot 2023-11-02 at 17 04 52

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Are any changes required to the e2e tests?
- [ ] If you've added a new route, have you added a new
  `auditEvent`? (see `server/routes/apply.ts` for examples)
- [ ] Are there environment variables or other infrastructure configuration which needs to be included in this release?
- [ ] Are there any data migrations required. Automatic or manual?
- [ ] Does this rely on changes being deployed to the CAS API?

## Post merge checklist

Once we've merged it will be auto-deployed to the dev environment.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui).

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

### CODE REVIEW

File: integration_tests/pages/page.ts [hunk 1]
```diff
@@ -79,4 +80,37 @@ export default abstract class Page {
   clickRemove(): void {
     cy.get('a').contains('Remove').click()
   }
+
+  removeWhiteSpaceAndLineBreaks(stringToReplace: string): string {
+    return stringToReplace.trim().replace(/(\r\n|\n|\r)/gm, '')
+  }
+
+  removeHtmlBreaks(stringToReplace: string): string {
+    return stringToReplace.replace(/<br \/>/g, '')
```
Comment (patrickjfl) [inline integration_tests/pages/page.ts:89]: what implications do these methods have for how we want to render ACCTs?

**Review by patrickjfl — APPROVED** [review]
Looks great, nice job!

File: integration_tests/pages/page.ts (hunk 1 above)
Comment (Harriethw) [reply integration_tests/pages/page.ts:89]: hopefully none - this is just for the integration tests to strip out characters/elements we don't need to check... if there are any further stylings or elements added we can strip them out as well.

Comment (patrickjfl) [reply integration_tests/pages/page.ts:89]: oh yeah, of course 🤦

File: integration_tests/tests/assess/view_submitted_application.cy.ts [hunk 2]
```diff
@@ -13,9 +13,10 @@ import { submittedApplicationFactory } from '../../../server/testutils/factories
 import paths from '../../../server/paths/assess'
 import Page from '../../pages/page'
 import SubmittedApplicationPage from '../../pages/assess/submittedApplicationPage'
+import { fullPersonFactory } from '../../../server/testutils/factories/person'
 
 context('Assessor views submitted application', () => {
-  const submittedApplication = submittedApplicationFactory.build()
+  // const submittedApplication = submittedApplicationFactory.build()s
```
Comment (edavey) [inline integration_tests/tests/assess/view_submitted_application.cy.ts:19]: did you mean to leave this in as commented out, or were you intending to remove it?

**Review by edavey — COMMENTED** [review]
👍 really clear. Just some v minor questions/suggestions...

**Pull request merged**